docs/data.md

```diff
@@ -9,6 +9,10 @@ This list does not include any `DEBUG` or `TRACE` level events, since those are
 not logged by default in production. The events below are grouped by crate, and
 the level and type of the log is listed.
 
+Any log containing sensitive data must include a boolean field `sensitive`
```

Comment:
Looks like we should add some tests to ensure this. Can be done in a separate follow-up.

Reply:
Ah, looks like it's already mentioned in the review comments above. I filed this [ticket](https://mozilla-hub.atlassian.net/browse/CONSVC-1590) to track it.

Reply:
I think it would be worth having someone from our team finish up this patch, since Jeff is having some above-and-beyond problems with getting Merino set up locally.

Reply:
I'm still going to be focused on incident response for the next bit too, so likely will be a while before I can set aside time to diagnose build issues.